setup.py

```diff
@@ -70,7 +70,6 @@
     "invenio-records-resources>=0.3.2,<0.4.0",
     "invenio-accounts>=1.3.0",
     "invenio-files-rest>=1.2.0",
-    "invenio-records-permissions>=0.8.0",
```

Comment:
Was this meant to be deleted? It is still used :) (I know it will be installed via invenio-records-resources, but that's perhaps too implicit and indirect yet...)

Reply:
fair enough, I took it out cuz it was coming from records-resources. Will put it back int.